chart/compass/values.yaml

```diff
@@ -41,4 +41,14 @@ global:
       namespace: compass-system
 
   database:
-    useEmbedded: true
\ No newline at end of file
+    useEmbedded: false
```

Comment:
useEmbedded should be still set to true IMO

Reply:
done